README.md

```diff
@@ -47,6 +47,22 @@ settings:
   # If label on the issue is not in specified list, this issue will be created as a Bug
   label_mapping:
     enhancement: Story
+
+  # (Optional) JIRA issue's summary
+  # This field can be used to customize the JIRA issue's summary (title).
+  # The value of the field will be passed to the python
+  # format() method to generate the JIRA summary. The GH issue
+  # variable will be captured in the format() method so that users can
+  # use GH issue attribute's values to build the JIRA summary.
+  # See GH issue definition :
+  # https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28
```

Comment:
I like this version better. My remaining concern that it is not obvious, that `issue` must be passed as a variable.

you mentioned "GH issue variable", would it be obvious that user has to use this attribute?
how do they know that this variable was predefined in the code without reading the code?

also, what will happen if we rename the variable in the code. Then we break the userland, right ?

Reply:
I propose to add a unit test to make sure that we do not break that

Reply:
@hector-cao we need both docs and unittests. Please add those :)

Reply:
i added customized summary field that makes use of the variable issue, if somehow, the summary evaluation fails, the tests will fail